bookkeeper-server/src/main/java/org/apache/bookkeeper/client/LedgerOpenOp.java

```diff
@@ -24,36 +24,40 @@
 import java.util.Arrays;
 import java.util.concurrent.TimeUnit;
 import java.security.GeneralSecurityException;
+import java.util.concurrent.CompletableFuture;
 
 import org.apache.bookkeeper.client.AsyncCallback.OpenCallback;
 import org.apache.bookkeeper.client.AsyncCallback.ReadLastConfirmedCallback;
 import org.apache.bookkeeper.client.BookKeeper.DigestType;
+import org.apache.bookkeeper.client.SyncCallbackUtils.SyncOpenCallback;
+import org.apache.bookkeeper.client.api.OpenBuilder;
 import org.apache.bookkeeper.proto.BookkeeperInternalCallbacks.GenericCallback;
 import org.apache.bookkeeper.stats.OpStatsLogger;
 import org.apache.bookkeeper.util.MathUtils;
 import org.apache.bookkeeper.util.OrderedSafeExecutor.OrderedSafeGenericCallback;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import org.apache.bookkeeper.client.api.ReadHandle;
 
 /**
  * Encapsulates the ledger open operation
  *
  */
-class LedgerOpenOp implements GenericCallback<LedgerMetadata> {
+class LedgerOpenOp implements GenericCallback<LedgerMetadata>  {
     static final Logger LOG = LoggerFactory.getLogger(LedgerOpenOp.class);
 
     final BookKeeper bk;
-    final long ledgerId;
```

Comment:
why do you remove 'final`

Reply:
it was from the previous version when the LedgerOpenOp was directly the builder.
Restored